x/programs/rust/wasmlanche-sdk/src/panic.rs

```diff
@@ -0,0 +1,12 @@
+pub fn register_panic() {
+    use crate::log;
```

Comment:
I think it would be better to put this function in the `log` module instead.

Reply:
I think that makes sense!